.travis.yml

```diff
@@ -6,21 +6,31 @@ branches:
     - master
     - module-9
     - module-10
+    - module-11
+    - module-12
+    - module-13
 
 script:
   - docker build --target test --tag my-test-image . 
   - docker run my-test-image tests
   - docker run --env SECRET_KEY --env MONGO_CLIENT my-test-image test_e2e
+install:
+  - wget https://releases.hashicorp.com/terraform/"$TF_VERSION"/terraform_"$TF_VERSION"_linux_amd64.zip
+  - unzip terraform_"$TF_VERSION"_linux_amd64.zip
+  - sudo mv terraform /usr/local/bin/
+  - rm terraform_"$TF_VERSION"_linux_amd64.zip
+  - terraform init 
 before_deploy:
+  - terraform apply -auto-approve -var 'CLIENT_ID'='c9d4bc4ddeb0af642b18' -var 'CLIENT_SECRET'=$CLIENT_SECRET -var "LOGGLY_TOKEN"=$LOGGLY_TOKEN -var 'SECRET_KEY'='secret-key' -var 'USER_ID'=$USER_ID
```

Comment:
Nice, I can see that Terraform is now deploying successfully as part of your pipeline 😄